architecture/platforms/versal_acap.rst

```diff
@@ -0,0 +1,125 @@
+.. _versal_acap:
+
+#############################################
+VERSAL Adaptive Compute Acceleration Platform
+#############################################
+
+
+.. _overview :
+
+Overview
+********
+
+The OP-TEE support for the AMD/Xilinx Versal Adaptive Compute Acceleration Platform (`Versal ACAP`_) delegates most of its functionality to the `PLM firmware`_ executing in the MicroBlaze processor.
```

Comment:
Need to briefly explain the client/server architecture.

Reply:
The principles of a mailbox driver is expected knowledge. I pointed to the server and the client. That is enough.

Reply:
It is not enough. Please add link to the TRM or something with more details as our code so customers have an idea of why the functionality is in the PLM.